packages/x-data-grid/src/tests/rowSelection.DataGrid.test.tsx

```diff
@@ -117,34 +116,36 @@ describe('<DataGrid /> - Row selection', () => {
       });
     });
 
-    it('should not select a range with shift pressed', () => {
-      render(<TestDataGridSelection />);
-      fireEvent.click(getCell(0, 0));
+    it('should not select a range with shift pressed', async () => {
+      const { user } = render(<TestDataGridSelection />);
+      await user.click(getCell(0, 0));
       expect(getSelectedRowIds()).to.deep.equal([0]);
-      fireEvent.click(getCell(2, 0), { shiftKey: true });
+      await user.keyboard('{Shift>}');
+      await user.click(getCell(2, 0));
+      await user.keyboard('{/Shift}');
       expect(getSelectedRowIds()).to.deep.equal([2]);
     });
   });
 
   describe('prop: checkboxSelection = false (single selection), with keyboard events', () => {
-    it('should select one row at a time on Shift + Space', () => {
-      render(<TestDataGridSelection disableRowSelectionOnClick />);
+    it('should select one row at a time on Shift + Space', async () => {
+      const { user } = render(<TestDataGridSelection disableRowSelectionOnClick />);
 
       const cell0 = getCell(0, 0);
-      fireUserEvent.mousePress(cell0);
-      fireEvent.keyDown(cell0, { key: ' ', shiftKey: true });
+      await user.click(cell0);
+      await user.keyboard('{Shift>}[Space/]{/Shift}');
```

Comment:
What does `/` in `[Space/]` do?

Reply:
So these 2 are equal?
`'{Shift>}[Space]{/Shift}'`
`'{Shift>}[Space/]{/Shift}'`

Reply:
Essentially, yes, they should be the same.
I'll check if I used both forms and align them accordingly. 👍

Reply:
Simplified to avoid usage of explicit `/` at the end when not necessary. 😉 
https://github.com/mui/mui-x/pull/15769/commits/4fa033ca5f69ef0589aaa9a04bb23fd1ff600f97